### DISCUSSION TITLE
Using pygments for syntax highlighting (not the pygments theme)

### DISCUSSION THREAD

**Discussion by alecloudenback** [jgm/pandoc#9337] [Q&A]
https://pygments.org/ has support for many more languages syntax highlighting than pandoc seems to support out of the box. Is there a way to use https://pygments.org/ for the highlighting? 

This came up because I was trying to get syntax highlighting for `julia-repl`.

**Comment by jgm** [comment]
We want everything to be self-contained in one executable; that means not depending on an installed python ecosystem.

But if you really want to use Pygments, you can do that by writing a Lua filter.

PS. If there are languages you'd like to see supported, see the jgm/skylighting repository and make a request there. There are KDE syntax definitions for languages we don't yet support (to keep the compile shorter), but they can be added. [but not julia-repl, sorry]

It is also possible to use a syntax definition that isn't directly compiled in to pandoc; see the documentation of `--syntax-definition`.  You might be able to modify the existing julia syntax definition to make a julia-repl syntax definition.
